commit 80ad2d01c5d15f17f243cca7b80517928c9137cf
Author: Narrat <autumn-wind@web.de>
Date:   Sat Mar 16 21:37:53 2024 +0100

    add xdata

diff --git a/src/package.h b/src/package.h
index 0a55975..4a6f732 100644
--- a/src/package.h
+++ b/src/package.h
@@ -32,7 +32,8 @@ enum pkg_entry {
     PKG_CHECKDEPENDS,
     PKG_FILES,
     PKG_BACKUP,
-    PKG_DELTAS
+    PKG_DELTAS,
+    PKG_XDATA
 };
 
 typedef struct pkg {
diff --git a/src/pkginfo.rl b/src/pkginfo.rl
index 909d8d7..d4daba1 100644
--- a/src/pkginfo.rl
+++ b/src/pkginfo.rl
@@ -43,7 +43,8 @@
            | 'optdepend'   %{ parser->entry = PKG_OPTDEPENDS; }
            | 'makedepend'  %{ parser->entry = PKG_MAKEDEPENDS; }
            | 'checkdepend' %{ parser->entry = PKG_CHECKDEPENDS; }
-           | 'backup'      %{ parser->entry = PKG_BACKUP; };
+           | 'backup'      %{ parser->entry = PKG_BACKUP; }
+           | 'xdata'       %{ parser->entry = PKG_XDATA; };
 
     entry = header ' = ' [^\n]* @store %emit '\n';
     comment = '#' [^\n]* '\n';
